client/src/components/GroupPage/GroupPage.css

```diff
@@ -0,0 +1,8 @@
+.wrapper {
+    text-align: center;
+}
+
+h1{
+    display: inline-block;
+}
+
```

Comment:
Can you use the css naming convention outlined in the readme? These styles will clash with styles throughout the rest of the app